tensorboard/defs/protos.bzl

```diff
@@ -13,8 +13,16 @@
 # limitations under the License.
 
 load("@protobuf//:protobuf.bzl", "py_proto_library")
+load("@protobuf//:protobuf.bzl", "cc_proto_library")
 
 def tb_proto_library(name, srcs = [], visibility = []):
+  cc_proto_library(
```

Comment:
Apparently, this call needs `default_runtime = "@protobuf//:protobuf",` for TensorBoard to build. ... yeah ... :/ I don't know exactly why.